### PR TITLE
GEOS-9188 image insert namespace support

### PR DESCRIPTION
**Pull request by NielsCharlier**

### CODE REVIEW

File: src/web/core/src/main/java/org/geoserver/web/wicket/CodeMirrorEditor.java [hunk 1]
```diff
@@ -121,9 +121,9 @@ public CodeMirrorEditor(String id, String mode, IModel<String> model) {
 
     public void addCustomButton(String title, String cssClass, CustomButtonAction action) {
         customButtons.add(
-                new AjaxLink<Object>(customButtons.newChildId()) {
+                new GeoServerAjaxFormLink(customButtons.newChildId()) {
```
[the remark this answers is not in the document]
Comment (NielsCharlier) [reply src/web/core/src/main/java/org/geoserver/web/wicket/CodeMirrorEditor.java]: I think you're right, I think this change has become obsolete

**Comment by NielsCharlier** [comment]
Thanks for your review @tbarsballe I have made the changes

**Comment by NielsCharlier** [comment]
ok to merge @tbarsballe ?

**Comment by NielsCharlier** [comment]
I assume from the comments this can be merged after the making the changes

**Comment by tbarsballe** [comment]
Sorry about the slow response @NielsCharlier - my Github notifications weren't making it to my inbox for some reason - fixed now.
I've got no objections to you merging (I see you have done so)

**Comment by NielsCharlier** [comment]
np thanks again for the review!

**Pull request merged**